src/api/flaskr/api/tts/aliyun_provider.py

```
@@ -0,0 +1,1401 @@
+                json=payload,
+                headers=headers,
+                timeout=60,
+            )
+
+            # Check content type to determine if error or audio
+            content_type = response.headers.get("Content-Type", "")
+
+            if (
+                "audio" in content_type
+                or response.status_code == 200
+                and "application/json" not in content_type
```

Ruff (0.14.10)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)